app/domain/permitted_globalized_attrs.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2012-2025, Puzzle ITC. This file is part of
+#  hitobito and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito.
+
+class PermittedGlobalizedAttrs
+  def permitted_attrs(original_permitted_attrs, entry)
+    @entry = entry
```

Comment:
Ich würde hier nicht in einem Methodencall eine Instanzvariable zuweisen, sondern entweder die lokale Variable weiterreichen, oder die Instanzvariable im initializer setzen.